src/DocumentFormat.OpenXml/Attributes/Translator/DocumentTagAttributeTranslator.cs

```diff
@@ -8,7 +8,7 @@ internal class DocumentTagAttributeTranslator : TagAttributeTranslator
         private static string[] arrayOfOrigTagAttrNames = { "conformance" };
         private static string[] arrayOfNewTagAttrNames = { "conformance" };
         private static string[] arrayOfOrigTagAttrValues = { "strict" };
-        private static string[] arrayOfNewTagAttrValues = { "" }; // Must be "".
+        private static string[] arrayOfNewTagAttrValues = { string.Empty }; // Must be "".
```

Comment:
can the comment be updated for clarity?

Reply:
... I have no idea why it 'must be ""'... I saw it while doing this change and thought about removing it because it doesn't seem to add any value

Reply:
... I have no idea why it 'must be ""'... I saw it while doing this change and thought about removing it because it doesn't seem to add any value

Reply:
I've just removed it. None of the other items have a comment about why it is necessary, so I'm under the assumption that they are all necessary.